### DISCUSSION TITLE
It is possible to read data for "Result beam" in API, RFEM 6?

### DISCUSSION THREAD

**Discussion by sonkaste1** [Dlubal-Software/Dlubal_CSharp_Client#57] [Q&A] [answered]
Hello,

I just ran across the Result Beam object in RFEM6.
https://www.dlubal.com/en/support-and-learning/support/knowledge-base/001406

It is possible to read properties and results for these entities using API?

Thank you!

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @sonkaste1 ,
you can read the properties of result beams with the function get_member(). Results should also be available for this type of member. You can access the results wit the normal result table functions (e.g. get_results_for_members_internal_forces()).
